CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # UNRELEASED
 
+-   Update matrix-rust-sdk to `#5d46b35d`, which includes:
+
+    -   Rename `UserIdentity` to `OtherUserIdentity` and
+        ([#4036](https://github.com/matrix-org/matrix-rust-sdk/pull/4036))
```

Comment:
and ... ?

To be honest, this feels like it buries the lede a bit. Setting this out as a change to matrix-rust-sdk makes it feel like an internal change that a user of the library doesn't need to worry about. This is a breaking change to the *js* API; the fact it's being done to stay in sync with the rust sdk feels barely here or there. Let's set it out as a breaking change in the same way we have with previous breaking changes.

Reply:
Ah, I didn't realise this was how we were laying it out. Fixed in f147076